oat-sa/tao-item-runner-qti-fe
# change textarea and ckeditor to div

Ticket: https://oat-sa.atlassian.net/browse/MS-446

**What has changed?**

At the moment, the `reviewRenderer` is rendering the extended text interaction using a `textarea` and `ckEditor` plugin. This is not good when the scorer would like to highlight the response of a test taker (which is rendered inside the `textarea` component or an `iframe` component) because you can not get the `range` of the text which is not part of the dom. To achieve the highlight feature for scorers in the reviewRenderer I have changed the textarea and ckEditor iframe with a simple div and the response is injected in this div (plain text for simple extended interactions and rich html for xhtml extended text interaction)

**How to test?**

- Check out the branch
- install this version of node modules in taoQtiItems Extention in your tao instance (https://medium.com/@jonchurch/use-github-branch-as-dependency-in-package-json-5eb609c81f1a)
- Create a test with extended text interaction in all three flavors (plain, preformatted, and XHTML)
- take the test on tao and login as a scorer on MS to review the responses of your test
-  All the extended text interactions should be working as expected and should have been rendered in a div instead of textarea and iframe
- Also run the tests: `npm run test`
